frontend_tests/node_tests/settings_user_groups.js

```diff
@@ -521,7 +521,7 @@ test_ui("reset", () => {
 });
 
 test_ui("on_events", ({override, mock_template}) => {
-    mock_template("confirm_dialog/confirm_delete_user.hbs", false, (data) => {
+    mock_template("dialog_widget/confirm_delete_user.hbs", false, (data) => {
```

Comment:
I think I'd prefer to skip this commit; even if the widget supports doing confirm_dialogs as well as other modals, logically/readability wise, it's nice to have `confirm_dialog` be its own contained thing.